README.md

```diff
@@ -40,6 +42,17 @@ Assume you're using debug build.
 For the advanced Zenoh configuration, you can either use `-h` to see more details or pass the configuration file with `-c`.
 The configuration file example is under the folder `config`.
 
-## Note
+## Troubleshooting
 
-The implementation follows the spec defined in [up-l1/zenoh](https://github.com/eclipse-uprotocol/up-spec/blob/main/up-l1/zenoh.adoc).
+1. Examples fail to communicate with each other.
+  
+   Check the multicast functionality of your network interface.
+   You can use the unicast connection directly.
+   For example:
+
+   ```shell
+   # server
+   ./target/debug/examples/rpc_server -l tcp/<IP>:<port>
+   # client
+   ./target/debug/examples/rpc_client -e tcp/<IP>:<port>
+   ```
```

Comment:
or we can suggest enabling multicast on the loopback
``` bash
$ sudo ip link set dev lo multicast on
```
or add the footnote https://github.com/eclipse-uprotocol/up-transport-zenoh-rust/pull/74/

Reply:
I'll close this one and adopt your PR. It's always better to have more contributors. :smile:

Reply:
either choice is fine with  me